core-services/prow/02_config/openshift/microshift/_prowconfig.yaml

```diff
@@ -50,6 +50,7 @@ tide:
     - main
     labels:
     - approved
+    - bugzilla/valid-bug
```

Comment:
Please remove MicroShift from the changes. Nothing in OCP depends on MicroShift.